docs/cmd/www/meroxa-resources-rotate-tunnel-key.md

```diff
@@ -20,7 +20,8 @@ meroxa resources rotate-tunnel-key NAME [flags]
 ### Options
 
 ```
-  -h, --help   help for rotate-tunnel-key
+  -f, --force   skip confirmation
```

Comment:
hm `--force` is really general for this specific case.. but i guess better than `--skip-confirmation`

Reply:
This has to do with the rotate keys command that I think came from this: https://github.com/meroxa/cli/pull/150

Reply:
I realize this comment should have been made in #150, but we should add the `buildCommandWithConfirm` function to the rotate-tunnel-key command to make sure that's there.

Reply:
@dianadoherty hope you don't mind I pushed this https://github.com/meroxa/cli/pull/152/commits/c6da44f93f2d5c52ff85d30cbc9979e26a1eee82